lib/utils.js

```diff
@@ -75,4 +75,44 @@ utils.parsePath = function(path, isPattern) {
     }
 };
 
+var reservedEncoderExpression
+        = /%(21|23|24|26|27|28|29|2A|2B|2C|2F|3A|3B|3D|3F|40|5B|5D)/ig;
```

Comment:
Readability would be improved if these followed the same ordering as the mappings below.